BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00319-03
\ No newline at end of file
+1.0.25-prerelease-00322-04
```

Comment:
`00322-04` is part of https://github.com/dotnet/corefx/pull/7989 which I'm merging when I get a clean Ubuntu Release CI run.